src/Sylius/Bundle/ApiBundle/Validator/Constraints/ChosenPaymentRequestActionEligibilityValidator.php

```diff
@@ -52,6 +52,10 @@ public function validate(mixed $value, Constraint $constraint): void
             return;
         }
 
+        if (null === $value->action) {
```

Comment:
It would be nice to have a PHPUnit test for this validator 🖖🏻

Reply:
I'm doing so right away 😉

Reply:
done!